src/gui/sdlmain.cpp

```diff
@@ -2494,6 +2499,9 @@ void Config_Add_SDL() {
 	Pint = Pmulti->GetSection()->Add_int("ysens",Property::Changeable::Always,100);
 	Pint->SetMinMax(-1000,1000);
 
+	Pbool = sdl_sec->Add_bool("rawmouseinput", always, false);
+	Pbool->Set_help("Mouse is routed through OS API by default. Enable this to poll mouse directly which bypasses OS mouse settings.");
+
```

Comment:
We have a mix of terminology between the variable versus description:
 - "raw true"
 - "raw false"
 - "OS API"
 - "Mouse direct polling"

Suggest we collapse these concepts into fewer terms, and use them in both the setting and the description:

```
mouse_movement = desktop` vs `direct` 
Pbool->Set_help("Mouse movements are provided by the Desktop Environment or polled directly without the influence of Desktop mouse settings")
```

Reply:
Absolutely no. "raw input" is a term gamers are pretty familiar with while "mouse movement desktop vs polled" is going to tell nothing to most people.